src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -166,11 +166,7 @@ const TABLE_FIELDS = {
         attributes: {
           className: 'text-left ml-1',
           showValueTooltip: true,
-          formatValue: value => (
-            <div className="d-flex">
-              {value?.translatedName ?? value?.name}
-              {renderHandlingIcons(value?.handlingIcons)}
-            </div>),
+          formatValue: value => getReceivingItemValue(value),
```

Comment:
in this case you can just pass the function
```suggestion
          formatValue: getReceivingItemValue,
```